GooglePlayInstant/Editor/PlayInstantQuickDeployWindow.cs

```diff
@@ -204,7 +205,14 @@ private void OnGuiLoadingScreenSelect()
 
             if (GUILayout.Button("Create Loading Scene", GUILayout.Width(ButtonWidth)))
             {
-                PlayInstantLoadingScreenGenerator.GenerateLoadingScreenScene();
+                if (string.IsNullOrEmpty(QuickDeployConfig.Config.assetBundleUrl))
+                {
+                    Debug.LogError("AssetBundle URL text field cannot be null or empty.");
```

Comment:
Would be preferable to display an error in addition to logging something to console. As it is, user clicks button and nothing visible happens. Could consider creating a helper method that does both, similar to PlayInstantBuilder. LogError()

Reply:
Resolved.